packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -342,12 +352,16 @@ class DatePicker extends React.Component {
     );
 
     return (
-      <div className={styles['date-picker']}>
-        <ResponsiveElement
-          responsiveTo="window"
-          defaultElement={portalPicker}
-          small={popupPicker}
-        />
+      <div className={cx(['container-div'])}>
```

Comment:
instead of container-div, I would like a more specific name, like date-picker-container. I would be ok with that one.

Reply:
date-picker-wrapper is ok as well, as another possible idea.

Reply:
date-picker-wrapper is a good one: https://github.com/cerner/terra-framework/pull/504/commits/2203fba9c165fddf755cb6251760768116a5b678